File: 06.LearnEspresso/LearnEspresso/app/src/main/java/com/dicoding/picodiploma/learnespresso/SecondActivity.java
package com.dicoding.picodiploma.learnespresso;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.view.MenuItem;
import android.widget.TextView;

public class SecondActivity extends AppCompatActivity {
    TextView tvResultView;
    public static String EXTRA_INPUT = "extra_input";

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_second);

        if (getSupportActionBar() != null) {
            getSupportActionBar().setTitle("Activity Second");
            getSupportActionBar().setDisplayHomeAsUpEnabled(true);
        }

        tvResultView = findViewById(R.id.tv_result_view);
        String input = getIntent().getStringExtra(EXTRA_INPUT);
        tvResultView.setText(input);
    }

    @Override
    public boolean onOptionsItemSelected(MenuItem item) {
        if (item.getItemId() == android.R.id.home) {
            finish();
        }
        return super.onOptionsItemSelected(item);
    }
}

File: 07.LearnEspresso/LearnEspresso/kotlin/app/src/main/java/com/dicoding/picodiploma/learnespresso/SecondActivity.kt
package com.dicoding.picodiploma.learnespresso

import android.os.Bundle
import android.view.MenuItem
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity

class SecondActivity : AppCompatActivity() {
    private lateinit var tvResultView: TextView

    companion object {
        var EXTRA_INPUT = "extra_input"
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_second)

        supportActionBar?.title = "Activity Second"
        supportActionBar?.setDisplayHomeAsUpEnabled(true)

        tvResultView = findViewById(R.id.tv_result_view)
        val input = intent.getStringExtra(EXTRA_INPUT)
        tvResultView.text = input
    }

    override fun onOptionsItemSelected(item: MenuItem): Boolean {
        when {
            item.itemId == android.R.id.home -> finish()
        }
        return super.onOptionsItemSelected(item)
    }
}
